dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/service/CompanyUploader.kt

```diff
@@ -126,4 +127,33 @@ class CompanyUploader(
             )
         }
     }
+
```

Comment:
My comments in the following might become irrelevant. As I understood the ticket this upload procedure should have been incorporated into the already existing requests done in the functions above to reduce traffic. I don't know if this was only about the one off upload and not the sunday updates but let's clarify this. I feel like the ticket description was not super clear here again.

Reply:
As clarified in today's daily this sequential implementation is perfectly fine.